Sources/MapboxMaps/Offline/TileRegionLoadOptions+MapboxMaps.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+extension TileRegionLoadOptions {
+    /// Initializes a `TileRegionLoadOptions`, required for
+    /// `TileStore.loadTileRegion(forId:loadOptions:)`
+    ///
+    /// - Parameters:
+    ///   - geometry: The tile region's associated geometry (optional).
+    ///   - descriptors: The tile region's tileset descriptors.
+    ///   - metadata: A custom JSON value to be associated with this tile region.
+    ///   - tileLoadOptions: Restrict the tile region load request to the
+    ///         specified network types. If none of the specified network types
+    ///         is available, the load request fails with an error.
+    ///   - averageBytesPerSecond: Limits the download speed of the tile region.
+    ///
+    /// `averageBytesPerSecond` is not a strict bandwidth limit, but only
+    /// limits the average download speed. tile regions may be temporarily
+    /// downloaded with higher speed, then pause downloading until the rolling
+    /// average has dropped below this value.
+    ///
+    /// If `metadata` is not a valid JSON object, then this initializer returns
+    /// `nil`.
+    public convenience init?(geometry: MBXGeometry?,
+                             descriptors: [TilesetDescriptor]?,
+                             metadata: Any? = nil,
```

Comment:
Did Android include default arguments for the parameters in these new methods?

Reply:
Android is not wrapping these APIs.

Reply:
So no default values, I guess?